Update Feature-Policy to Permissions-Policy (#22439)
mdn/content: files/en-us/web/api/featurepolicy/index.md

{{APIRef("Feature Policy")}}{{SeeCompatTable}}

> The `FeaturePolicy` interface represents the set of policies applied to the current execution context.

Reviewer: I'm not sure if the "of the XXX API" is mandated. But I do think that it is worth doing something like:
```suggestion
The `FeaturePolicy` interface represents the set of [permission policies](ADD LINK HERE) applied to the current execution context.
```

Author: I fixed this one in my next commit.

> ## FeaturePolicy Methods

Reviewer: ```suggestion
## Instance methods
```

- {{DOMxRef("FeaturePolicy.allowsFeature")}} {{Experimental_Inline}}
  - : Returns a Boolean that indicates whether or not a particular feature is enabled in the specified context.
- {{DOMxRef("FeaturePolicy.features")}} {{Experimental_Inline}}

>   - : Returns a list of names of all features supported by the User Agent. Features whose names appear on the list might not be allowed by the Permissions Policy of the current execution context and/or might not be accessible because of users' permissions.

Reviewer: ```suggestion
  - : Returns a list of names of all features supported by the User Agent. Features whose names appear on the list might not be allowed by the Permissions Policy of the current execution context and/or might be restricted by user-granted permissions.
```
